Makefile

```diff
@@ -0,0 +1,19 @@
+# This is the installation directory under the container
+TORCH_INSTALL_DIR=/home/drgscl/app/torch/distro/install
+
+PREFIX = $(TORCH_INSTALL_DIR)
+CFLAGS = -I$(PREFIX)/include/THC -I$(PREFIX)/include/TH -I$(PREFIX)/include \
+	-I$(PWD)/third_party `pkg-config --cflags opencv`
```

Comment:
Added `-I$(PWD)/third_party` which contains `png++`. It is a header-only library, so we don't have to compile it.